trace_test.go

```diff
@@ -82,6 +86,35 @@ func TestTraceExporter(t *testing.T) {
 			t.Fatalf("expected to flush 1, got %d", len(me.payloads()))
 		}
 	})
+
+	t.Run("sampler", func(t *testing.T) {
+		eq := equalFunc(t)
+		me := newTestTraceExporter(t)
+		me.exportSpan(spanPairs["error"].oc)
+
+		time.Sleep(time.Millisecond) // wait for recv
```

Comment:
Seeing this, and seeing it's repeating here https://github.com/DataDog/opencensus-go-exporter-datadog/pull/27/files#diff-25469609881ec3304d4084de8b1aea8fR82 it looks like there's a pattern of calling `time.Sleep(time.Millisecond)` before `stop()`. I think it's fine for the test, but what I'm thinking is that, maybe, users of the library would, generally speaking, expect:
```go
exporter.ExportSpan(span)
exporter.Stop()
```
to actually report the span. It's only about waiting a millisecond, but I think it's worth it, OK it will make stop 1 msec slower and not even 100% reliable as the span "could" stay inflight somewhere in the channel (which is 5e5 big, so not blocking for sure). I'll let you decide it, because honestly I don't know which is best, illusion of transactionality and shortest testing code (what I propose, adding a `time.Sleep` just before https://github.com/DataDog/opencensus-go-exporter-datadog/blob/7f668c590c2bf7351be9a23aff6c057f6440338c/trace.go#L140) or faster execution but little tweaks in tests because it has an akward behavior (what we have now).

Reply:
You're right. I really don't like having `Sleep` in tests. I find it semi-unacceptable. To be honest however, I find it even more unacceptable to have it in actual code. For that reason, I've pushed another commit that drains the input channel at the time `Stop` is called. Would you be happy with this solution? (check last commit)